hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/subscription/health/SubscriptionHealthContextCreator.java

```diff
@@ -0,0 +1,31 @@
+package pl.allegro.tech.hermes.management.domain.subscription.health;
+
+import pl.allegro.tech.hermes.api.Subscription;
+
+import static java.lang.Double.parseDouble;
+
+class SubscriptionHealthContextCreator {
```

Comment:
Is in this package any class name conflict? if no then all qualified names from it can by replaced by imports.